experimental/LieAlgebras/src/serialization.jl

```diff
@@ -130,6 +130,116 @@ function load_type_params(s::DeserializerState, ::Type{<:LieAlgebraElem})
   return load_typed_object(s)
 end
 
+###############################################################################
+#
+#   Lie algebra modules
+#
+###############################################################################
+
+@register_serialization_type LieAlgebraModule uses_id [:_dummy_attribute]
```

Comment:
for now maybe just a more meaningful name?
```suggestion
@register_serialization_type LieAlgebraModule uses_id [:serialize_construction]
```